### PR TITLE
Added Generate and Calculate Median Operations

### PR DESCRIPTION
**Pull request by AbderraoufDrine**

### CODE REVIEW

File: metricstics/utils/operation.py
Comment (MajorTreble) [inline metricstics/utils/operation.py:1]: Please keep the files in src/

File: metricstics/utils/operation.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+def sort(data):
```
Comment (MajorTreble) [inline metricstics/utils/operation.py:1]: What kind of sort is this?

File: metricstics/src/controller/input_controller.py [hunk 2]
```diff
@@ -21,7 +22,7 @@ def generate_random_data(self, size):
         Arg:
             size(int): the length of the data
         """
-        self.data = {1, 2, 3, 4, size}
+        self.data = [random.uniform(0, 1000) for _ in range(size)]
```
Comment (MajorTreble) [inline metricstics/src/controller/input_controller.py:25]: This should have a wider range

File: metricstics/utils/operation.py (hunk 1 above)
Comment (AbderraoufDrine) [reply metricstics/utils/operation.py:1]: this is bubble sort

**Review by MajorTreble — APPROVED** [review]

**Pull request merged**